backend/docs/example_process_miner_config.yaml

```
@@ -33,3 +33,59 @@ tags:
       'get_transactions':
         - 'GET_TRANSACTIONS'
         - 'get transaction list'
+  label:
+    source: 'message'
+    tag_all: false
+    defalt_value: '_else_'
```

This should be `default_value`
```suggestion
    default_value: '_else_'
```